.github/workflows/secrets-sdk.yml

```diff
@@ -0,0 +1,275 @@
+name: Secrets SDK CI
+env:
+  DEBUG: napi:*
+  APP_NAME: keyring
+  MACOSX_DEPLOYMENT_TARGET: "10.13"
+"on":
+  push:
+    paths:
+      - "packages/secrets/**"
+      - ".github/workflows/secrets-sdk.yml"
+  pull_request:
+    paths:
+      - "packages/secrets/**"
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: packages/secrets
```

Comment:
This `defaults` section could be moved up above `jobs`, so that we only have to define it once for all jobs, rather than twice inside each job (`build` and `test`).